src/main/CMakeLists.txt

```diff
@@ -1,4 +1,4 @@
-# GNSS-SDR is a Global Navigation Satellite System software-defined receiver.
+# gnss-sdr is a Global Navigation Satellite System software-defined receiver.
```

Comment:
This needs to be capital letters. Please avoid changing comments in CMakeLists.txt files if possible